app/serializers/curated_list_serializer.rb

```diff
@@ -14,4 +14,17 @@ def to_json(*_args)
       }
     end)
   end
+
+  def light_to_json
+    JSON.generate(@curated_lists.map do |curated_list|
+      {
+        "name" => curated_list.name,
+        "description" => curated_list.description,
+        "slug" => curated_list.slug,
+        "articles" => curated_list.articles.map do |article|
+          article.id
+        end
+      }
+    end)
+  end
```

Comment:
Note for later: we can probably utilize this lighter method on the readingList component as well. To optimize it, need to eager load the articles in the initial ActiveRecord query at the controller level of each.